deploy/deploy-operator-go/main.go

```diff
@@ -15,248 +16,285 @@ import (
+		if excludeCrds {
+			// Download file
+			filePath, cleanup, err := downloadFile("https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/v1.8.0/deploy/crds/pulumi.com_stacks.yaml")
+			if err != nil {
+				return err
+			}
+			crds, err := yaml.NewConfigFile(ctx, "crds", &yaml.ConfigFileArgs{
+				File: filePath,
+			})
+			if err != nil {
+				return err
+			}
+			defer cleanup()
+
+			deploymentOpts = append(deploymentOpts, pulumi.DependsOn([]pulumi.Resource{crds}))
 		}
-		operatorRole, err := rbacv1.NewRole(ctx, "operator-role", &rbacv1.RoleArgs{
-			Rules: rbacv1.PolicyRuleArray{
-				&rbacv1.PolicyRuleArgs{
-					ApiGroups: pulumi.StringArray{
-						pulumi.String(""),
-					},
-					Resources: pulumi.StringArray{
-						pulumi.String("pods"),
-						pulumi.String("services"),
-						pulumi.String("services/finalizers"),
-						pulumi.String("endpoints"),
-						pulumi.String("persistentvolumeclaims"),
-						pulumi.String("events"),
-						pulumi.String("configmaps"),
-						pulumi.String("secrets"),
-					},
-					Verbs: pulumi.StringArray{
-						pulumi.String("create"),
-						pulumi.String("delete"),
-						pulumi.String("get"),
-						pulumi.String("list"),
-						pulumi.String("patch"),
-						pulumi.String("update"),
-						pulumi.String("watch"),
-					},
-				},
-				&rbacv1.PolicyRuleArgs{
-					ApiGroups: pulumi.StringArray{
-						pulumi.String("apps"),
-					},
-					Resources: pulumi.StringArray{
-						pulumi.String("deployments"),
-						pulumi.String("daemonsets"),
-						pulumi.String("replicasets"),
-						pulumi.String("statefulsets"),
-					},
-					Verbs: pulumi.StringArray{
-						pulumi.String("create"),
-						pulumi.String("delete"),
-						pulumi.String("get"),
-						pulumi.String("list"),
-						pulumi.String("patch"),
-						pulumi.String("update"),
-						pulumi.String("watch"),
-					},
-				},
-				&rbacv1.PolicyRuleArgs{
-					ApiGroups: pulumi.StringArray{
-						pulumi.String("monitoring.coreos.com"),
-					},
-					Resources: pulumi.StringArray{
-						pulumi.String("servicemonitors"),
-					},
-					Verbs: pulumi.StringArray{
-						pulumi.String("create"),
-						pulumi.String("get"),
-					},
-				},
-				&rbacv1.PolicyRuleArgs{
-					ApiGroups: pulumi.StringArray{
-						pulumi.String("apps"),
-					},
-					ResourceNames: pulumi.StringArray{
-						pulumi.String("pulumi-kubernetes-operator"),
-					},
-					Resources: pulumi.StringArray{
-						pulumi.String("deployments/finalizers"),
-					},
-					Verbs: pulumi.StringArray{
-						pulumi.String("update"),
+
+		for _, ns := range namespaces {
+			operatorServiceAccount, err := corev1.NewServiceAccount(ctx, fmt.Sprintf("operator-service-account-%s", ns), &corev1.ServiceAccountArgs{
+				Metadata: &metav1.ObjectMetaArgs{
+					Namespace: pulumi.String(ns),
+					Annotations: pulumi.StringMap{
+						"pulumi.com/skipAwait": pulumi.String("true"), // workaround per https://github.com/pulumi/pulumi-kubernetes/issues/2063
```

Comment:
The linked issue is closed, so this may not be needed anymore.

Reply:
I needed it at the time of writing; I can try bumping the pulumi/kubernetes version in go.mod and see if that helps.

Reply:
(bumped the version of the provider in go.mod)